integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/predicate/BoolSearchPredicateIT.java

```diff
@@ -406,6 +406,41 @@ public void nested() {
 				.hasDocRefHitsAnyOrder( INDEX_NAME, DOCUMENT_1 );
 	}
 
+	@Test
+	public void nested_withConstantScore() {
```

Comment:
Should be named `withConstantScore`. You're not using the `nested` feature here.